ui/app/clinical/displaycontrols/treatmentData/directives/treatmentTable.js

```diff
@@ -2,7 +2,9 @@
 
 angular.module('bahmni.clinical')
     .directive('treatmentTable', function () {
-        var controller = function ($scope, $rootScope) {
+        var controller = function ($scope, $rootScope, treatmentService, messagingService) {
+            $scope.prescriptionEmailToggle = $rootScope.prescriptionEmailToggle;
+            $scope.prescriptionSMSToggle = $rootScope.prescriptionSMSToggle;
```

Comment:
why not pass it as attribute to the directive? instead of reading from rootScope

Reply:
passed as attributes to directive